README.md

```diff
@@ -27,7 +27,7 @@ talks_to = [ "consul","postgres" ]
 
 
 ```
-% ./dependencies -sisu ~/sandbox/git/work/sisu -service actionrequest-service -region eu -environment stg
+% ./dependencies deploy-order ~/sandbox/git/work/sisu stg eu -service actionrequest-service
```

Comment:
would be nice to use full stage names (eg: `sisu-sandbox-eu`) rather than having to pass the individual parts (also because there will be environment specific dependencies), ie:

Reply:
The way I see it, It's really just a matter of the user input you receive, I don't understand the big complication about it :)
You can still break down the user input into separate parts in the application code and keep the logic as is.

user input: `sisu-sandbox-eu`
parts: `{ env = sandbox, project = sisu, region = eu }`

Reply:
Yes, I could split it and interpret that part X is Y.
What would be the advantage of having a single argument for it?